torch/csrc/autograd/python_engine.cpp

```diff
@@ -172,6 +171,8 @@ PyObject *THPEngine_run_backward(THPEngine *self, PyObject *args, PyObject *kwar
     THPObjectPtr py_outputs {PyTuple_New(num_inputs)};
     if (!py_outputs) return nullptr;
     for (int i = 0; i < num_inputs; i++) {
+      THPUtils_assert(allow_unreachable || outputs[i].defined(), "One of the differentiated "
```

Comment:
Can you please change the error message to suggest using the flag if that's what someone wants?